bc_obps/registration/fixtures/mock/facility_designated_operation_timeline.json

```diff
@@ -232,5 +232,185 @@
       "operation": "1bd04128-d070-4d3a-940a-0874c4956181",
       "start_date": "2024-06-28T23:20:02Z"
     }
+  },
```

Comment:
We either need to update both timeline fixtures to match the transfer_event fixture, or change that to match this

Reply:
I suggest prioritizing the population of timeline tables first, as they are more critical. The transfer_event fixtures can be handled afterward as the final step.